Typo & wording fix - Fargate vs EC2 (#13405)
pulumi/docs: content/blog/fargate-vs-ec2/index.md

Ok, EC2 costs less than Fargate on a pure cost-of-compute basis. If my t3.medium ($0.0416/hr) can typically run 6-8 pods reliably, and this works out to under $0.01/hr per pod. Comparable Fargate pods (0.5 vCPU/1GB) cost about $0.025/hr each, so 2.5x.

> These calculations can change, based on how much headroom you need on EC2 and how many containers you can run on a give EC2 instance, but Fargate is just more expensive unless all your containers exactly fit into the compute sizes offered by Fargate. And AWS is likely incuring costs maintaining warm compute capacity for launching Fargate pods, so it shouldn't be suprizing that Fargate is more expensive.

Reviewer: an idea to rephrase a bit for clarity
```suggestion
These calculations can change based on how much headroom you need on EC2 and how many containers you can run on a given EC2 instance. Still, Fargate is just more expensive unless all your containers fit the exact computing sizes offered by Fargate. And don't forget, AWS is likely incurring costs maintaining the warm compute capacity for launching Fargate pods, so it shouldn't be surprising that Fargate is more expensive.
```
